controls/radofficenavigationbar/overflow-menu.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Overflow Menu
+page_title: Overflow Menu
+description: Check our &quot;Overflow Menu&quot; documentation article for the RadOfficeNavigationBar control.
+slug: radofficenavigationbar-overflow-menu
+tags: radofficenavigationbar, overflow menu, overflow
+publushed: True
+position: 8
+---
+
+# Overflow Menu
+
+When the number of the `RadNavigationBarItem` elements exceeds the count of the maximum visible items, they are placed in the overflow menu. To review the overflow menu, click on the navigation options button.
+
+>important The items in the overflow menu do not support drag and drop operations. 
```

Comment:
Almost the entire article is created by notes. Notes are useful if you have a lot of content and you want to highlight something. I think that the content here is not that much. You can add these as separate paragraphs. In other words, you can remove the note tags (important and tip) and leave the plain text.